packages/training/python/src/training/babylon_env.py

```
@@ -280,32 +325,56 @@ async def setup(self):
+        reader = HuggingFaceTrajectoryReader(config)
+        await reader.connect()
+        
+        # Get trajectory groups in the same format as database loading
+        self.trajectory_cache = reader.get_trajectory_groups(
+            min_agents_per_window=self.config.min_agents_per_window
         )
-        logger.info("Initialized EvaluationSuite and RolloutDumper")
+        
+        # Log stats
+        stats = reader.get_stats()
+        logger.info(f"HuggingFace dataset stats:")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)